src/Diagnostics.Scripts/ApiBlockConfig.json

```diff
@@ -1,14 +1,24 @@
 {
   "PInvokeBlockConfig": [
     {
-      "BlockMatchingRegExPattern": "^(?!dnsapi.dll\\b).*",
+      "BlockMatchingRegExPattern": "^(?!(dnsapi.dll|dnsapi)\\b).*",
       "MessageToShowWhenBlocked": "",
       "MethodsToBlock": [
         {
           "BlockMatchingRegExPattern": "(.*)",
           "MessageToShowWhenBlocked": "PInvoke calls are prohibited."
         }
       ]
+    },
+    {
+      "BlockMatchingRegExPattern": "^(dnsapi.dll|dnsapi)\\b",
+      "MessageToShowWhenBlocked": "",
+      "MethodsToBlock": [
+        {
+          "BlockMatchingRegExPattern": "^(?!(DnsQuery_W|DnsRecordListFree)\\b).*",
```

Comment:
if I understand this correctly, this means block all methods that does not have DnsQuery_W or DnsRecordListFree??

Reply:
Yes, that is correct. Block call calls other than DnsQuery_W and DnsRecordListFree from dnsapi.dll